docs/studio/cli/project.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Project
+description: Manage and inspect information about your Studio projects.
+keywords:
+  - CLI
+  - command line
+  - studio
+---
+
+## `studio project <subcommand>`
+
+Use these commands to interact with your Studio projects, including listing projects associated with your accounts, creating new projects, and adding members to teams that manage the project.
+
+### `ls [identifier]`
+
+List all of your projects tied to your Studio account, optionally, passing the team ID. Defaults to your Studio session context.
+
+| Argument       | Type     | Default                | Description                         |
+| -------------- | -------- | ---------------------- | ----------------------------------- |
+| `[identifier]` | `string` | Studio session default | (Optional) the team's ID, if known. |
+
+### `create <name> <description>`
+
+Create a new project by defining the project's name and description.
+
+| Argument        | Type     | Description                |
+| --------------- | -------- | -------------------------- |
+| `<name>`        | `string` | The project's name.        |
+| `<description>` | `string` | The project's description. |
+
+You must also include the team ID for which the project is being created for.
+
+| Option     | Type     | Default | Description                                            |
+| ---------- | -------- | ------- | ------------------------------------------------------ |
+| `--teamId` | `string` | --      | The unique team ID (via the `studio team ls` command). |
+
+### `add [team] [user]`
+
```

Comment:
similar to the `team add` command, i couldnt get this one to work. i tried the same type of variations as those mentioned in that comment.